### DISCUSSION TITLE
does this work with OpenVPN?

### DISCUSSION THREAD

**Discussion by REDDDDY** [Mygod/VPNHotspot#385] [General]
I have an HDMI encoder hardware unit (for IRL Stream) that has wifi but very limited in configuration on its webgui interface.
My phone is already rooted and installed with VPNHotspot.
-managed to get the encoder connected to the phone via wlan1 (monitored) through 192.168.174.133/24 and has its own ip address 
at 192.168.174.10.

This phone is connected to home VPN server using OpenVPN which has its default subnet at 10.8.0.0/24

Is it possible for the hdmi encoder to communicate to devices that are within the LAN resources (through this OpenVPN)

**Comment by Mygod** [comment]
Last time I checked, yes via NAT.

**Comment by REDDDDY** [comment]
Can you help explain a bit on NAT.
is this on the VPN Server side ? or the smartphone side?
